### PR TITLE
Adds powder that tastes like powder and some others

### PR DESCRIPTION
**Pull request by ActualRory**
:cl:
add: Three new prank-orientated powders.
/:cl:

Thanks to:
https://github.com/HippieStation/HippieStation/pull/3637 - Yes.

### CODE REVIEW

**Comment by ActualRory** [comment]
It's okay.
:sob:

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
be fucking consistent

**Comment by ActualRory** [comment]
Be less of a poop-head.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
come back im sorry

**Comment by ActualRory** [comment]
I knew you'd come crawling back.
:heart_eyes:

**Comment by ActualRory** [comment]
Fuck your consistency.
:wink:

**Comment by Partheo** [comment]
It lives!

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -340,3 +340,54 @@
...
+	if(prob(1))
+		M.say("This tastes nothing like powder, I have no idea what I expected.")
+	..()
+
+/datum/reagent/drug/honkpowder
+	name = "Honk Powder"
+	id = "honkpowder"
+	description = "This is powder? Are you joking?"
+	color = "#ff5bff" //Neon Pink
+	reagent_state = LIQUID
```
Comment (Partheo) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:381]: Liquid? Ree.

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -340,3 +340,54 @@
...
+	description = "This is powder? Doesn't taste like it."
+	color = "#d6d6d6"
+	reagent_state = LIQUID
+
+/datum/reagent/drug/tastepowder/on_mob_life(mob/living/M)
+	var/high_message = pick("This couldn't taste more powdery if it tried.", "Oh boy, powder flavour!", "I can't believe it's not powder!")
+	if(prob(5))
+		M.say("[high_message]")
+	if(prob(20))
+		M.say("This tastes exactly like powder.") //Sometime before this point powder stopped feeling like a real word. - Rory.
```
Comment (Partheo) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:371]: Powder is actually not a real word

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm:381]: thatsthejoke.dll?

**Comment by Alek2ander** [comment]
:hankey:

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 3]
```diff
@@ -53,3 +53,25 @@
 	result = "bolamine"
 	required_reagents = list("space_drugs" = 2, "frostoil" = 1)
 	result_amount = 2
+
+
+ /datum/chemical_reaction/yespowder
+ 	name = "Yes Powder"
+ 	id = "yespowder"
+ 	result = "yespowder"
+ 	required_reagents = list("itching_powder" = 1, "morphine" = 1)
```
Comment (Amelia0010) [inline code/modules/reagents/chemistry/recipes/drugs.dm:62]: How exactly do you make "itching_powder"?

Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/reagents/chemistry/recipes/drugs.dm:62]: /datum/chemical_reaction/itching_powder
    name = "Itching Powder"
    id = "itching_powder"
    result = "itching_powder"
    required_reagents = list("welding_fuel" = 1, "ammonia" = 1, "charcoal" = 1)

Comment (Amelia0010) [reply code/modules/reagents/chemistry/recipes/drugs.dm:62]: i'm dumb, ignore me

Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/reagents/chemistry/recipes/drugs.dm:62]: You might be dumb but you are my dumb 💙

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 4]
```diff
@@ -265,6 +265,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 /obj/item/clothing/mask/cigarette/attack(mob/living/carbon/M, mob/living/carbon/user)
 	if(!istype(M))
 		return ..()
+	if(M.on_fire && !lit)
+		light("<span class='notice'>[user] lights [src] with [M]'s burning body. What a cold-blooded badass.</span>")
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline code/game/objects/items/weapons/cigs_lighters.dm]: atomize your prs

Comment (ActualRory) [reply code/game/objects/items/weapons/cigs_lighters.dm]: Fuck, good catch. Not sure I ended up commiting that to this. Must have fumbled that one in when I switched from GitDesktop to GitKraken, branches got a tad tangled.

Comment (ActualRory) [reply code/game/objects/items/weapons/cigs_lighters.dm]: Should be sorted now.

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 5]
```diff
@@ -340,3 +340,54 @@
...
+	name = "Yes Powder"
+	id = "yespowder"
+	description = "Powder that makes you say yes."
+	color = "#fffae0"
+	reagent_state = LIQUID
+
+/datum/reagent/drug/yespowder/on_mob_life(mob/living/M)
+	var/high_message = pick("Agreement fills your mind.", "'No' is so last year. 'Yes' is in.", "Yes.")
+	if(prob(5))
+		M << "<span class='notice'>[high_message]</span>"
```
Comment (AshCorr) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: You should use to_chat here.
`to_chat(M, "<span class='notice'>[high_message]</span>")`

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 6]
```diff
@@ -53,3 +53,25 @@
 	result = "bolamine"
 	required_reagents = list("space_drugs" = 2, "frostoil" = 1)
 	result_amount = 2
+
```
Comment (AshCorr) [inline code/modules/reagents/chemistry/recipes/drugs.dm:56]: Relax on that enter button.

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 7]
```diff
@@ -340,3 +340,54 @@
...
+/datum/reagent/drug/honkpowder/on_mob_life(mob/living/M)
+	var/high_message = pick("Henk.", "HENK!!", "HONK!!!!!")
+	if(prob(30))
+		M.say("[high_message]")
+	if(prob(40))
+		M.say("Honk.")
+	if(prob(1))
+		M.say("END MY SUFFERING.")
+	if(prob(5))
+		M << "<span class='notice'>Your funny bone itches.</span>"
```
Comment (AshCorr) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: use to_chat here again.

**Pull request merged**